packages/node_modules/samples/browser-multi-party-call/app.js

```diff
@@ -32,11 +32,19 @@ let activeMeeting, webex;
 // Webex and start listening for incoming calls, so we'll wrap a few things
 // up in a function.
 function connect() {
+  const services = document.getElementById('production').checked ? {} : {
+    discovery: {
+      u2c: 'https://u2c-intb.ciscospark.com/u2c/api/v1',
+      hydra: 'https://apialpha.ciscospark.com/v1/'
+    }
+  };
+
   return new Promise((resolve) => {
     if (!webex) {
       // eslint-disable-next-line no-multi-assign
       webex = window.webex = Webex.init({
         config: {
+          services,
```

Comment:
You can simplify this and inline it

```suggestion
          ...(!document.getElementById("production").checked && {
	      services: {
	        discovery: {
	          u2c: "https://u2c-intb.ciscospark.com/u2c/api/v1",
	          hydra: "https://apialpha.ciscospark.com/v1/",
	        },
	      },
	    })
```